src/main/java/adris/altoclef/util/baritone/InteractWithBlockPositionProcess.java

```diff
@@ -2,95 +2,121 @@
 // Source code recreated from a .class file by IntelliJ IDEA
 // (powered by FernFlower decompiler)
 //
+// ... bruh
```

Comment:
lmfao oops

Reply:
The sources for this are public afaik

Reply:
yep, I got a little lost in the sauce there.